test/integration/crud/bulk.test.js

```diff
@@ -22,6 +27,76 @@ describe('Bulk', function () {
   before(function () {
     return setupDatabase(this.configuration);
   });
+  describe('BulkOperationBase', () => {
+    describe('#raw', function () {
+      let client;
+      beforeEach(async function () {
+        client = this.configuration.newClient();
+        await client.connect();
+      });
+      afterEach(async function () {
+        await client.close();
+      });
+      context('when called with an undefined operation', function () {
+        it('should throw a MongoInvalidArgument error ', async function () {
+          try {
+            client.db('test').collection('test').initializeUnorderedBulkOp().raw(undefined);
+            expect.fail('Failure to throw error');
+          } catch (error) {
+            expect(error).to.be.instanceOf(MongoInvalidArgumentError);
+          }
```

Comment:
```suggestion
          const bulkOp = client.db('test').collection('test').initializeUnorderedBulkOp()
          let error = await bulkOp.raw(undefined).catch(error => error);
          expect(error).to.be.instanceOf(MongoInvalidArgumentError);
          error = await bulkOp.raw(true).catch(error => error);
          expect(error).to.be.instanceOf(MongoInvalidArgumentError);
          error = await bulkOp.raw(3).catch(error => error);
          expect(error).to.be.instanceOf(MongoInvalidArgumentError);
```

This syntax is just a preference but it makes it a bit cleaner to add additional coverage here, I'm throwing in a number and boolean to hit that non-object case I suggested